src/anemoi/training/config/lam.yaml

```diff
@@ -0,0 +1,36 @@
+defaults:
+- data: zarr
+- dataloader: native_grid
+- diagnostics: evaluation
+- hardware: example
+- graph: limited_area
+- model: graphtransformer
+- training: default
+- _self_
+
+
+### This file is for local experimentation.
+##  When you commit your changes, assign the new features and keywords
+##  to the correct defaults.
+# For example to change from default GPU count:
+# hardware:
+#   num_gpus_per_node: 1
+
+dataloader:
+  dataset:
+    cutout:
+      - dataset: ${hardware.paths.data}/${hardware.files.dataset}
+        thinning: ???
+      - dataset: ${hardware.paths.data}/${hardware.files.forcing_dataset}
+    adjust: all
+    min_distance_km: 0
+#  spatial_indices:
+#    _target_: anemoi.training.utils.spatial_indices.GraphNodeAttribute
+#    nodes_name: data
+#    node_attribute_name: indices_connected_nodes
+model:
+  output_mask: cutout_mask
+hardware:
```

Comment:
I am worried about overpopulating the folder. We plan to have global, limited_area, stretched default configs. And we will have the debug, and hierarchical as options for each of the defaults. 

For the CI and PRs, it would be nice if we could set a --debug (or similar) that overrides the num_channels, batch_size, and steps. Not sure if this is possible, but it would solve the problem without adding more configs.

Reply:
Yes i think you're right about keeping a balance with the configs we have in develop. The suggestion for the CI with a debug flag could be nice! @theissenhelen  do you have a view if that feature would be easy to implement?